docs/eventing/channels/channels-crds.md

```diff
@@ -6,7 +6,8 @@ type: "docs"
 ---
 
 # Available Channels
-
+<!-- Move this content out to the README.md so that it's part of a conceptual overview of Channels.
+It's not helpful for content this brief and this specific to have its own page. -->
```

Comment:
I disagree, I think it will get unwieldy for a list of all available channels to live in the README, also I think this is more of a reference doc than something that belongs in the main page. Why not just link to it instead of dumping it all back into one file?

Reply:
Also, as mentioned in the comment below I think this is a generated file, so idk how nicely that would play in README